tests/src/Kernel/Paragraphs/LinksBlockTest.php

```diff
@@ -0,0 +1,350 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\Paragraphs;
+
+use Drupal\paragraphs\Entity\Paragraph;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the "Links block" and "Social media follow" paragraphs.
+ */
+class LinksBlockTest extends ParagraphsTestBase {
+
+  /**
+   * Tests the rendering of the paragraph type.
+   */
+  public function testRendering(): void {
+    // Create social media follow paragraph with horizontal variant.
+    $paragraph = Paragraph::create([
+      'type' => 'oe_social_media_follow',
+      'field_oe_title' => 'Social media title',
+      'field_oe_social_media_variant' => 'horizontal',
```

Comment:
Since we added fields that are meant to be shown in the UI, we need to also write functional tests that check that the correct fields are in place.